example/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "cumulus-integration-tests",
   "version": "1.0.0",
   "description": "Cumulus Integration Test Deployment",
+  "private": true,
```

Comment:
This seems like a good thing. Is it there to prevent Lerna from treating this folder as a package?

Reply:
This change lets lerna treat `example` as a package, and link  dependencies into it, but ensures that we'll never deploy the example package to npm.